src/main/java/mate/academy/springboot/swagger/util/SortUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortUtil {
```

Comment:
```suggestion
@Component
public class SortUtil {
```
use it as field in controller